sup3r/bias/bias_correct_means.py

```diff
@@ -0,0 +1,949 @@
+"""Classes to compute means from vortex and era data and compute bias
+correction factors.
```

Comment:
Do you want to include any details on where / how to download the vortex means? No code just a quick tip.